File: BonAppetit.py
ab=input().split()
n=int(ab[0])
k=int(ab[1])
c=input().split()
charge=int(input())
sum=0
for i in c:
    sum+=int(i)
sum-=int(c[k])
if (sum/2)==charge:
    print("Bon Appetit")
else:
    print(charge-int((sum/2)))

File: BonAppetit.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        int k=sc.nextInt();
        int arr[]=new int[n];
        for(int i=0;i<n;i++)
            arr[i]=sc.nextInt();
        int charge=sc.nextInt();
        int total=0;
        for(int i=0;i<n;i++)
            if(i!=k)total+=arr[i];
        if(total/2<charge){
            System.out.println(charge-(total/2));
        }
        else{
            System.out.println("Bon Appetit");
        }
    }
}